### DISCUSSION TITLE
I can't find the Function ID option

### DISCUSSION THREAD

**Discussion by RenaKunisaki** [NationalSecurityAgency/ghidra#2796] [Q&A] [answered]
I loaded an fidb, but under Auto Analyze, there's no Function ID option. It was there for another program. What's going on?

**Comment by Wall-AF** [comment, marked as answer]
Seems you need to close and re-open the Code Browser.

**Reply by RenaKunisaki** [reply]
Seems like I had to actually restart Ghidra completely this time. I'd already closed and reopened the Code Browser. (Maybe because another Code Browser was open to a different program?)

**Reply by ryu-highabusa** [reply]
This still happens in 10.3 in 2023 so I'm glad I found this thread. Was going nuts thinking a key feature had been removed. Thanks y'all.
